package.json

```
@@ -1,20 +1,60 @@
 {
-  "name": "css-frameworks-ca",
-  "version": "1.0.0",
-  "description": "Test",
+  "name": "workflow",
+  "version": "0.4.2",
```

The version should never go backwards
```suggestion
  "version": "1.4.2",
```